src/answerfile.cpp

```diff
@@ -313,8 +314,24 @@ void AnswerFile::loadNVHPC()
     m_tools.emplace_back(std::make_shared<NVhpc>());
 }
 
+void AnswerFile::loadServices() { loadFail2ban(); }
+
+void AnswerFile::loadFail2ban()
+{
+    if (!checkEnabled("fail2ban")) {
+        return;
+    }
+
+    m_services.emplace_back(std::make_shared<fail2ban>());
```

Comment:
I don't see this pointer being used in more than one location. Why is it a `shared_ptr`?